pauthabielf64/pauthabielf64.rst

```diff
@@ -3,7 +3,7 @@
    CC-BY-SA-4.0 AND Apache-Patent-License
    See LICENSE file for details
 
-.. |release| replace:: 0.1
+.. |release| replace:: 0.2
 .. |date-of-issue| replace:: 1\ :sup:`st` October 2020
 .. |copyright-date| replace:: 2020
 
```

Comment:
I see I forgot to change the footer markup in this doc. I'll do that myself at some point.